### DISCUSSION TITLE
https://mvnrepository.com/artifact/MrPowers

### DISCUSSION THREAD

**Discussion by DzmitryZhurykTR** [mrpowers-io/spark-fast-tests#85] [General]
Hi Matthew,

Could you upload the last scala 2.12 for spark 3 releases on https://mvnrepository.com/artifact/MrPowers?
I find it in jar by link https://repo1.maven.org/maven2/com/github/mrpowers/spark-fast-tests_2.12/ and it will be great to also have in mvnrepo.

Thanks.

**Comment by MrPowers** [comment]
@DzmitryZhurykTR - thanks for starting this discussion.  I used to publish packages with Spark Packages and they'd upload the JAR files to https://mvnrepository.com/artifact/MrPowers. 

Spark Packages [has unfortunately been broken for many years](https://github.com/databricks/sbt-spark-package/issues/31) at this point.  I needed to shift away from Spark Packages because it didn't work.

I think mvnrepository.com is just a mirror of what's in repo1.maven.org.  Looks like [the new JAR files are mirrored here in MVNRepository](https://mvnrepository.com/artifact/com.github.mrpowers/spark-fast-tests_2.12).

Thanks again for starting the discussion!

**Reply by DzmitryZhurykTR** [reply]
Got it. Thank you for the fast reply!
